internal/app/go-filecoin/storage_market_connector/client.go

```diff
@@ -0,0 +1,229 @@
+package storagemarketconnector
+
+import (
+	"bytes"
+	"context"
+	"time"
+
+	cborutil "github.com/filecoin-project/go-cbor-util"
```

Comment:
Call this something else to make it clear it is not gfc's internal cborutil but encoding functions that belong to external datatypes: perhaps `smcborutil`